sources/packages/backend/libs/integrations/src/services/disbursement-schedule-service/integration-disbursement-schedule-service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from "@nestjs/common";
+import { RecordDataModelService, DisbursementSchedule } from "@sims/sims-db";
+
+/**
+ * Service layer for Student Application disbursement schedules.
+ */
+@Injectable()
+export class IntegrationDisbursementSchedulerService extends RecordDataModelService<DisbursementSchedule> {
```

Comment:
It seems that this class should be named `IntegrationDisbursementScheduleService`, right?